traitsui/tests/editors/test_button_editor.py

```diff
@@ -33,6 +36,12 @@ class ButtonTextEdit(HasTraits):
 
     play_button_label = Str("I'm a play button")
 
+    play_button_image = Image(
+        ImageResource(
+            name="run", search_path='traitsui/examples/demos/Advanced/images'
```

Comment:
This _may_ be fragile because this is a relative path, so if the tests are invoked from an unexpected place you might not find what you expect.

Probably better to have a copy local to the test module (eg. in `tests/editors/images` or something similar)